### DISCUSSION TITLE
FairByDesign

### DISCUSSION THREAD

**Discussion by alaninmcr** [seek4science/seek#745] [Ideas]
For discussion of relationship of SEEK to https://www.fairbydesign.nl

**Comment by alaninmcr** [comment]
Carole says:

Fairbydesign is definitely to include in the FAIRDOM suite. Not just for plants

**Comment by floradanna** [comment]
DataHub has been developing the features of the Metadata Configurator within SEEK. Together with Custom Metadata, most of the features of the Metadata Configurator can be available in SEEK as well:  Metadata Configurator --> SEEK
- Project, Investigation, Study Information --> Custom Metadata
- Observation Unit Information --> Study Sample types derived from Study Templates (= ISA Source, Sample collection protocol, ISA Sample)
- Sample Information --> Assay Sample type derived from Assay Template (= ISA Samples and OtherMaterial)
- Assay Information --> Assay Sample type derived from Assay Template (=ISA Data File)

Additional explanation here: https://github.com/seek4science/seek/issues/143
